rest/revocation_test.go

```diff
@@ -2350,26 +2340,22 @@ func TestRevocationNoRev(t *testing.T) {
 		revocationTester.fillToSeq(4)
 		version := rt.PutDoc(docID, `{"channels": "A"}`)
 
-		require.NoError(t, rt.WaitForPendingChanges())
-		firstOneShotSinceSeq := rt.GetDocumentSequence("doc")
-
 		// OneShot pull to grab doc
-		err := btcRunner.StartOneshotPull(btc.id)
-		assert.NoError(t, err)
+		rt.WaitForPendingChanges()
+		btcRunner.StartOneshotPull(btc.id)
 
 		_ = btcRunner.WaitForVersion(btc.id, docID, version)
+		firstOneShotSinceSeq := rt.GetDocumentSequence("doc")
```

Comment:
There are a few spots where you're moving the firstOneShotSinceSeq sequence lookup to happen after StartOneShotPull?  It seems innocuous (the pull shouldn't be modifying the doc sequence on the server) - just curious whether this was required/intentional.